api/src/main/java/io/strimzi/api/kafka/model/KafkaAuthorizationKeycloak.java

```diff
@@ -44,6 +45,7 @@ public class KafkaAuthorizationKeycloak extends KafkaAuthorization {
     private Integer connectTimeoutSeconds;
     private Integer readTimeoutSeconds;
     private List<String> superUsers;
+    private boolean enableMetrics = false;
```

Comment:
Stupid question ... how does this work? The `enableMetrics` enables the metrics mechanism inside the OAuth libraries in server. And then the user has to also configure the export rules in the `Kafka` CR to tell the Prometheus JMX exporter to export these JMX metrics as Prometheus metrics. Is that correct?

Reply:
That is correct. There's always a possibility or concern that metrics collection may result in greater memory usage or poorer performance so it has to be enabled explicitly. For Prometheus JMX Exporter it integrates via the same mechanism as other metrics. I'm preparing an example OAuth enabled + metrics Kafka cluster deployment, which I'm still testing.

Reply:
Ok, thanks for the explanation. Yeah, I think it makes sense -> just wanted to confirm my understanding is right.